app/views/group_plan_summary.scala.html

```diff
@@ -14,74 +14,106 @@
  * limitations under the License.
  *@
 
-@import utils.ERSUtil
+@import _root_.utils.ERSUtil
 @import config.ApplicationConfig
-@import utils.CountryCodes
+@import _root_.utils.CountryCodes
 @import uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+@import views.html.templates.reference
+@import views.html.govuk_wrapper
 
-@this(form: FormWithCSRF)
+@this(
+		govuk_wrapper: govuk_wrapper,
+		govukBackLink: GovukBackLink,
+		govukSummaryList: GovukSummaryList,
+		govukButton: GovukButton,
+		hmrcPageHeading: HmrcPageHeading,
+		formWithCSRF: FormWithCSRF
+)
 
 @(requestObject: RequestObject, groupSchemeType: String, companies: CompanyDetailsList)(implicit request: Request[AnyRef], flash: Flash, messages: Messages, countryCodes: CountryCodes, ersUtil: ERSUtil, appConfig: ApplicationConfig)
-@* Is a group scheme template *@
+
 @schemeId = @{requestObject.getSchemeId}
-@import views.html.templates._
 
-@ers_main(title = ersUtil.getPageElement(schemeId, "ers", "group_summary.page_title") + " " + messages("ers.title.postfix")) {
+@govuk_wrapper(title = ersUtil.getPageElement(schemeId, "ers", "group_summary.page_title") + " " + messages("ers.title.postfix")) {
```

Comment:
(I hope this is the right view) axe is finding issues here: one thing with the badly ordered `ol`s, the other about the 'Change' button only reading out 'Change' rather than 'Change (company name)'

Reply:
Ah yes, will update now.